src/changelog.ts

```diff
@@ -161,27 +159,30 @@ export default class Changelog {
           .map(ref => ref.substr(TAG_PREFIX.length));
       }
 
-      const issueNumber = findPullRequestId(message);
-
       return {
         commitSHA: sha,
         message,
         // Note: Only merge commits or commits referencing an issue / PR
         // will be kept in the changelog.
         tags: tagsInCommit,
-        issueNumber,
+        issueNumber: null,
```

Comment:
if we're always setting this to null shouldn't that mean we need to update the type and remove it. I can see you are using the commitSHA below so can we just delete this line and remove it from the CommitInfo type?

Reply:
It will be set later

Reply:
mm, I see, its not used in the end